context/app/static/js/shared-styles/alerts/OutlinedAlert.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { Alert } from './index';
+
+export default {
+  title: 'Alerts/OutlinedAlert',
+  component: Alert,
+  argTypes: {
+    severity: {
+      options: ['warning', 'error', 'success', 'info'],
+      control: { type: 'select' },
+    },
+  },
+};
+
+const Template = (args) => <Alert {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  severity: 'warning',
+  children: 'Alert, intruders!',
+};
```

Comment:
Will an epilogue like this be standard for us? Just skimming the doc, should we be trying to follow naming conventions so [single story hoisting](https://storybook.js.org/docs/riot/writing-stories/naming-components-and-hierarchy#single-story-hoisting) applies?

Reply:
Fixed relevant stories.